Tests/ttLib/tables/tables_test.py

```diff
@@ -1,327 +1,298 @@
-        "aots/gsub_context1_boundary_f2.otf":			('GDEF','GSUB'),
-        "aots/gsub_context1_expansion_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context1_lookupflag_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context1_lookupflag_f2.otf":			('GDEF','GSUB'),
-        "aots/gsub_context1_multiple_subrules_f1.otf":		('GDEF','GSUB'),
-        "aots/gsub_context1_multiple_subrules_f2.otf":		('GDEF','GSUB'),
-        "aots/gsub_context1_next_glyph_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context1_simple_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context1_simple_f2.otf":			('GDEF','GSUB'),
-        "aots/gsub_context1_successive_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context2_boundary_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context2_boundary_f2.otf":			('GDEF','GSUB'),
-        "aots/gsub_context2_classes_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context2_classes_f2.otf":			('GDEF','GSUB'),
-        "aots/gsub_context2_expansion_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context2_lookupflag_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context2_lookupflag_f2.otf":			('GDEF','GSUB'),
-        "aots/gsub_context2_multiple_subrules_f1.otf":		('GDEF','GSUB'),
-        "aots/gsub_context2_multiple_subrules_f2.otf":		('GDEF','GSUB'),
-        "aots/gsub_context2_next_glyph_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context2_simple_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context2_simple_f2.otf":			('GDEF','GSUB'),
-        "aots/gsub_context2_successive_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context3_boundary_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context3_boundary_f2.otf":			('GDEF','GSUB'),
-        "aots/gsub_context3_lookupflag_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context3_lookupflag_f2.otf":			('GDEF','GSUB'),
-        "aots/gsub_context3_next_glyph_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context3_simple_f1.otf":			('GDEF','GSUB'),
-        "aots/gsub_context3_successive_f1.otf":			('GDEF','GSUB'),
-        "aots/lookupflag_ignore_attach_f1.otf":			('GDEF','GSUB'),
-        "aots/lookupflag_ignore_base_f1.otf":			('GDEF','GSUB'),
-        "aots/lookupflag_ignore_combination_f1.otf":		('GDEF','GSUB'),
-        "aots/lookupflag_ignore_ligatures_f1.otf":		('GDEF','GSUB'),
-        "aots/lookupflag_ignore_marks_f1.otf":			('GDEF','GSUB'),
-    ]
 
-    def __init__(self, methodName):
-        unittest.TestCase.__init__(self, methodName)
-        # Python 3 renamed assertRaisesRegexp to assertRaisesRegex,
-        # and fires deprecation warnings if a program uses the old name.
-        if not hasattr(self, "assertRaisesRegex"):
-            self.assertRaisesRegex = self.assertRaisesRegexp
+ttLibVersion_RE = re.compile(r' ttLibVersion=".*"')
 
-    def setUp(self):
-        pass
 
-    def tearDown(self):
-        pass
+def getpath(testfile):
+    path = os.path.dirname(__file__)
+    return os.path.join(path, "data", testfile)
```

Comment:
it doesn't look like it's needed. Maybe the `os` module is the right thing when joining the paths.
At least Appveyor didn't complain about it (you can see the number of failing tests is the same as on Travis):

https://ci.appveyor.com/project/fonttools/fonttools/build/1.0.459/job/knqaovpde5a65g37

I think `os.path.normpath` would be the best way to do that.